packages/react-components/lib/map/label-marker.tsx

```diff
@@ -177,7 +177,7 @@ export function LabelContainer(props: LabelContainerProps): JSX.Element | null {
           height={labelLocation.borderBBox.maxY - labelLocation.borderBBox.minY}
           rx={contentBorderRadius}
           ry={contentBorderRadius}
-          fill={theme.palette.background.paper}
+          fill={theme.palette.info.dark}
```

Comment:
Can you explain the rational for changing the color? I feel the info color makes it pop out too much and doesn't look good.

Reply:
![image](https://user-images.githubusercontent.com/28668944/140002264-6245f695-3e99-4534-81d4-44967ecf2b53.png)
oh because in light mode there was a conflict and I thought it was too subtle in dark mode

but ok I can update the text color instead, so it will look like this
![image](https://user-images.githubusercontent.com/28668944/140002516-3303f138-8249-4875-947c-8e848b212de6.png)

![image](https://user-images.githubusercontent.com/28668944/140002582-f4666ae2-74d8-4dc7-bff9-f40666f1aa22.png)

is this better?

Reply:
Seems that the label text color should be fixed to use the theme colors, the light theme should have black text and the dark theme should have white text.

Reply:
@vallq I notice something weird, the label border and text should be using the primary color, but in the buggy screenshot above, the label text seems to be using the background color. Maybe something messed up with the styles.

On another note, since the label text is using primary color, it might blend in too much with the background for the rmf theme, if so then there is a problem with the theme and we should look at a different primary color.